src/components/screens/home-screen.tsx

```diff
@@ -54,11 +57,39 @@ const HomeScreen = () => {
           content
         )}
       </Grid>
-        </Grid>
-      );
+    </Grid>
+  );
+
+  /**
+   * Renders the questionnaire card
+   * 
+   * @returns ReactNode containing the questionnaire card
+   */
+  const renderQuestionnaireCard = () => (
+    <Link to={adminMode ? "/admin/questionnaire" : "/questionnaire"} style={{ textDecoration: "none" }}>
+      <Card
+        sx={{
+          "&:hover": {
+            background: "#efefef"
+          }
+        }}
+      >
+        <CardContent>
+          <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
+            {strings.questionnaireProgress?.title || "Questionnaires"}
+          </Typography>
+          <Grid container>
+            <Box sx={{ width: "100%", display: "flex", flexDirection: "column" }}>
+              <QuestionnaireProgress />
+            </Box>
+          </Grid>
+        </CardContent>
+      </Card>
+    </Link>
+  );
```

Comment:
We already have the QuestionnaireCard, shouldn't this be done in there rather than here. All other cards render the card content in the file related to the card.